controllers/airflowcluster_controller.go

```diff
@@ -623,6 +623,18 @@ func gitContainer(s *alpha1.GitSpec, volName string) (bool, corev1.Container) {
 			{Name: "GIT_SYNC_USERNAME", Value: s.User},
 		}...)
 	}
+	if s.VerifySsl != nil {
+		if *s.VerifySsl == false {
+			env = append(env, []corev1.EnvVar{
+				{Name: "GIT_SSL_NO_VERIFY", Value: "true"},
+			}...)
+		}
+	}
+	if s.SubmoduleMode != "" {
+		env = append(env, []corev1.EnvVar{
+			{Name: "GIT_SYNC_SUBMODULES", Value: s.SubmoduleMode},
```

Comment:
*Weird edge case*
The yaml format spec converts `off` to a boolean false which causes problem in `AirflowCluster` custom resource

```
dags:
  git:
    # yaml processor will interpret off as boolean false which golang will convert to string "false"
    submodules: off
```

This means that you will need to check if `s.SubmoduleMode == "false"` and then set `Value` to the string "off".

Reply:
RESOLVED: The current `git-sync` image used in this version of airflow does not support submodules. This code was removed